rosflight_sim/src/sil_board.cpp

```diff
@@ -70,6 +70,16 @@ void SILBoard::gazebo_setup(gazebo::physics::LinkPtr link, gazebo::physics::Worl
   auto remote_host = node_->get_parameter_or<std::string>("ROS_host", "localhost");
   int remote_port = node_->get_parameter_or<int>("ROS_port", 14520);
 
+  // IMU timer
```

Comment:
Are you intending to leave these comments here? That's fine if sure, just wanted to make sure you didn't forget them

Reply:
Hahaha nope thats where I was going to put them.